v5/core/utils.go

```diff
@@ -255,3 +256,18 @@ func SliceContains(slice []string, contains string) bool {
 	}
 	return false
 }
+
+// return the value of query parameter `param` from url, or "" when not found
+func GetQueryParam(urlStr *string, param string) (string, error) {
+	if IsNil(urlStr) {
+		return "", nil
+	}
+
+	u, err := url.Parse(*urlStr)
+	if err != nil {
+		return "", err
+	}
+
+	q := u.Query()
```

Comment:
One main difference that you'll notice in the methods I linked to above is the use of the url's `RawQuery` field and calling the `ParseQuery()` method.  I don't remember the significance right now, but there must have been a reason I coded it like that - LOL.

Reply:
I think this could make a difference if a) there is no query string, or b) if the query string exists but is in some way invalid.  I'll try to write some tests for these cases to see how my implementation handles them.

Reply:
Thanks for the feedback Phil. I've modified `GetQueryParam` to be more like the Platform Services methods, though a bit more concise (style preference) and updated the tests to cover all the cases (I think).